### PR TITLE
MDE/PKFE-7

### PR DESCRIPTION
**Pull request by mantvydasdeltuva**
- [MDE/PKFE-7 implemented toolbar groups selector](https://github.com/Strexas/kath/commit/186261d7a8f2e0714885082196260c1e16cb52de)
- [MDE/PKFE-7 implemented toolbar group](https://github.com/Strexas/kath/commit/ddeca20a4313939ba675fbea18c427b4db485a02)
- [MDE/PKFE-7 exporting components](https://github.com/Strexas/kath/commit/ebc56c7af66fe19f048a3bb58d4943578ecf578f)
- [MDE/PKFE-7 implemented toolbar view](https://github.com/Strexas/kath/commit/a014ab36869ce73a5251e96f53a2681e58d83993)

### CODE REVIEW

File: app/front-end/src/features/editor/components/toolbarView/toolbarGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    </List>
+  );
+};
+
+export interface ToolbarGroupItemProps {
+  group: string;
+  icon: SvgIconComponent;
+  label: string;
+  onClick: () => void;
+}
```
Comment (justinnas) [inline app/front-end/src/features/editor/components/toolbarView/toolbarGroup.tsx:35]: Interface ToolbarGroupitemProps and ToolbarGroup could be seperated into seperate files

File: app/front-end/src/features/editor/components/toolbarView/toolbarGroupsSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+export const ToolbarGroupsSelector: React.FC<ToolbarGroupsSelectorProps> = ({ children }) => {
+  return <List sx={{ height: '25%', p: '0', overflow: 'auto' }}>{children}</List>;
+};
+
+export interface ToolbarGroupsSelectorItemProps {
+  id: string;
+  label: string;
+  onClick: () => void;
+  groupRef?: string;
+}
```
Comment (justinnas) [inline app/front-end/src/features/editor/components/toolbarView/toolbarGroupsSelector.tsx:16]: Exports could be seperated into different files

**Review by justinnas — CHANGES_REQUESTED** [review]
Minor changes needed

**Comment by mantvydasdeltuva** [comment]
- [MDE/PKFE-7 component refactoring and added jsdocs](https://github.com/Strexas/kath/pull/37/commits/8edd32b6d20e48a3f98e7a7b9c22d89253466380)

File: app/front-end/src/features/editor/components/toolbarView/toolbarGroup.tsx (hunk 1 above)
Comment (mantvydasdeltuva) [reply app/front-end/src/features/editor/components/toolbarView/toolbarGroup.tsx]: Seperated `jsx` components into different files. Interfaces are kept as close as possible to the `jsx` component.

File: app/front-end/src/features/editor/components/toolbarView/toolbarGroupsSelector.tsx (hunk 2 above)
Comment (mantvydasdeltuva) [reply app/front-end/src/features/editor/components/toolbarView/toolbarGroupsSelector.tsx]: Same response

**Review by justinnas — APPROVED** [review]
Great work!

**Pull request merged**